### PR TITLE
Mile stone #4 : Add quotes generator

### PR DESCRIPTION
**Pull request by ahmed025566**
In this milestone you could see that I have made the following; 

- I have created a component that called Fetch in this component I have made the following : 

- I have created some states and its modifiers to store some data at them later in the component.
- Then I have fetched the data and stored the quote and the author in two states data and author.
- Then I have created a catch statement to catch errors and stored the error in a state called error.
- Later in the component I check if the loading gives true then I will display the loading statement if not It will display the content itself.
- finally I have made an element that check if the error state is empty then it will not display anything and if there an error it will display the error.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @ahmed025566,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### The live demo link in your readme header is broken. Kindly remove it or create live demo section and add a deployed link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/API.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import './API.css';
+
+function Fetch() {
```
Comment (fobadara) [inline src/components/API.js]: [REQUIRED] One of the general principles of coding is descriptive naming. Kindly name this component including the filename `Quotes`

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @ahmed025566, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good use of try-and-catch block
✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @ahmed025566, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good use of try-and-catch block
✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @ahmed025566**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- A <Quotes> component has been created. ✔️  
- The external API [API Ninjas](https://api-ninjas.com/api/quotes) has been used correctly. ✔️  
- The useEffect hook has been used correctly. ✔️  
- There is a loading state and an error state. :heavy_check_mark: 
- The `<Quotes>` component is imported into the main component, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**